Api/src/controllers/officeController.js

```diff
@@ -0,0 +1,73 @@
+import {
+  v4 as uuidv4
+} from 'uuid';
+import express from 'express';
+
+let offices = [{
+  id: uuidv4(),
+  "type": "Federal Lawmaker",
+  "name": "Senate"
+}];
+
+import dataFile from "../data.json";
+
+export const getOffices = async (req, res, next) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'application/json');
+  res.send(JSON.parse(JSON.stringify(dataFile)));
+  next();
```

Comment:
does `res.send` followed by `next()` work? it doesn't seem to make sense. in express you either respond or you next, not both. pls verify this. I hope you have tested the functionality and it works without problems on postman, because I'm only doing code-review right now, and I'm sure this won't work correctly.

Reply:
and when you check, remember that you are doing that in many places

Reply:
Done.